src/pages/cli/project/troubleshooting.mdx

```diff
@@ -0,0 +1,263 @@
+2. Deleted user pools created by Amplify from the Cognito console, or
+3. Updated the code in Lambda functions which were originally deployed through Amplify.
+These types of unmanaged changes will introduce a drift for the affected resources. Therefore subsequent updates to these resources using Amplify will most likely fail deployment.
+
+**Suggested Resolution:** 
+These types of drifts must be addressed on a case-by-case basis. In some cases, such as a Lambda code change, forcing a redeployment of the function by making a minor change to the code will likely work. 
+In other cases, like deleting user-pools or updates to GraphQL schema from the AppSync console, follow the steps below:
+1. Forcible deployment of CloudFormation stack from the CLI - Run `amplify push --force` to force push the application’s resource stack to be deployed.
+1. Forcible deployment of CloudFormation stack from the Console - Fetch the CloudFormation stack from the application’s deployment bucket and selectively run the resource specific part of the stack through the CloudFormation console.
+1. Apply drift changes into code: If both the above methods to deploy CloudFormation have failed for you, the other option is to inspect the manual changes and to update them in the CLI input files. 
+For example, you could update the GraphQL schema to reflect the same changes as performed manually in the cloud and then perform `amplify push`.
+
+### Push failures in a multi-account multi-environment project (CI/CD errors)
+<Callout>
+
+**Errors:** 
+
+"There was an error initializing your environment.”  
+"!!! Build failed"
+"!!! Non-Zero Exit Code detected"
+</Callout>
+
+In a multi-account project with CI/CD, you find that the application has deployed successfuly in one account but has failed deployment in another account. 
+The following are some common causes:
+1. Some resource being referenced in the Amplify generated CloudFormation is missing from the failing account. 
+1. The failing account is missing valid permissions to access a particular resource used in the CloudFormation. 
+1. The application code relies on “secrets” and secrets are not correctly created or copied in the failing environment.
+
+#### Scenario 1: "amplify push" fails in one account/environment
+Assume you created a project in a “dev” account and you extend your deployment environment to a full CI/CD pipeline with dev, beta and prod environments. 
+Your project likely follows a pattern specified in the [Amplify CLI team environments guide](/cli/teams/overview/#setting-up-prod-and-dev-environments). 
+On completion of all steps you observe that Amplify Hosting console displays the “build” step of one of your stages ("beta") as failed and shows other stages as successfully deployed.
+
+**Suggested Resolution:** 
+Using Amplify CLI deploy the application in the failing account. In the above case, dev stage has successfully deployed and beta stage has failed. 
```

Comment:
"Using Amplify CLI deploy the application in the failing account. " I don't understand this part.

Reply:
```suggestion
Using Amplify CLI deploy the application in the failing account. In the above case, the dev environment has successfully deployed but the beta environment has failed. 
```